lib/docker.ts

```diff
@@ -14,6 +14,16 @@ interface DockerOptions {
   tlsKey?: string;
 }
 
+interface Globs {
+  manifestGlobs?: string[];
+  binariesGlobs?: string[];
```

Comment:
since we're using singular form for ```manifest``` maybe we should also use it for ```binary```? ```binaryGlobs```? does it make sense? or more likely is my English just bad? 😬

Reply:
yes. fixed